TKGame/BackEnd/EntityManager.cs

```diff
@@ -136,13 +139,44 @@ public static void DrawHealth(SpriteBatch spriteBatch, Entity entity)
         /// </summary>
         public static void DamageEnemy()
         {
-            foreach (Entity entity in entities)
+            int count = 0;
+            foreach(Entity entity in entities)
             {
-                if (entities[0].entityTexture == Art.PlayerTexture) //Checks if player has sword out
+                if(entity.isEnemy && entity.hitBox.Intersects(entities[0].weapon.hitbox))
                 {
-                    if (entities[0].weapon.hitbox.Intersects(entity.hitBox) && entity != entities[0]) //Checks if hitboxes intersect and is not the player
+                    if (count % 240 == 0)
                     {
-                        entity.health -= (int)(entities[0].weapon.damageStat);
+                        if (entity == entities[1])
+                        {
+                            entities[1].health -= entities[0].weapon.damageStat;
+                        }
+                        else if(entity == entities[2])
+                        {
+                            entities[2].health -= entities[0].weapon.damageStat;
+                        }
+                        else if(entity == entities[3])
+                        {
+                            entities[3].health -= entities[0].weapon.damageStat;
+                        }
+
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Checks if the player hitbox intersects with any enemy hitboxes and then decrements player health
+        /// </summary>
+        public static void EnemyDamage()
+        {
+            int count = 0;
+            foreach(Entity entity in entities)
+            {
+                if ( entity.hitBox.Intersects(entities[0].hitBox) && entity.isEnemy)
+                {
+                    if (count % 240 == 0)
```

Comment:
this is a very hacky method of doing this. We can't guarantee 240 frames per sec and I don't see count get incremented anywhere so 0 % 240 = 240 is always the case